taskcluster/kinds/module-build/kind.yml

```diff
@@ -38,19 +37,6 @@ task-defaults:
       - ':{module_name}:assembleRelease'
       - ':{module_name}:publish'
       - ':{module_name}:checkMavenArtifacts'
-    dummy-secrets:
-        by-level:
-            '3': []
-            default:
-                - content: "faketoken"
-                  path: .symbols_upload_token
-    secrets:
-        by-level:
-            '3':
-                - name: project/application-services/symbols-token
-                  key: token
-                  path: .symbols_upload_token
-            default: []
```

Comment:
This is a cleanup I noticed while looking at this; since module-build doesn't upload symbols, it doesn't need access to the symbols.m.o upload token, only upload-symbols does.

Reply:
Ahh, I see.  That makes total sense.